TalismanSociety/talisman-web
# fix: stake sheet amount error validation after submission

# Description

Fixes Amount input validation error "insufficient funds" when an extrinsic is being submitted.

## Type of change


### 🧪 **How to test:**
- Visit `/staking/providers`
- Choose an asset to stake
- Click approve and sign transaction
- No error in Amount input should be displayed while waiting for the transaction to be mined

### 🖼️ **Screenshots:**


https://github.com/user-attachments/assets/9343d7a6-a572-40af-a421-df37085ab709



closes https://github.com/TalismanSociety/talisman-web/issues/1184
